packages/core/src/components/date-input/date-input.tsx

```diff
@@ -38,6 +37,12 @@ import {
 } from '../utils/input';
 import { makeRef } from '../utils/make-ref';
 import type { DateInputValidityState } from './date-input.types';
+import {
+  closeDropdown,
+  createValidityState,
+  handleIconClick,
+  openDropdown,
```

Comment:
Maybe we can use an alias to not have the name openDropdown twice in the same scope. Also time-input